lib/Sema/TypeCheckAttr.cpp

```diff
@@ -3274,6 +3274,8 @@ void AttributeChecker::visitDifferentiableAttr(DifferentiableAttr *attr) {
     return;
   }
 
+  if (!attr->getOriginalFunction())
```

Comment:
I think this should never happen. Could you just assert `attr->getOriginalFunction()` instead of setting it?

Reply:
Done in https://github.com/apple/swift/pull/27985/commits/12bd5680c2e41573761a62648d63185327b2df4e, thanks!